src/Microsoft.AspNetCore.Mvc.ViewFeatures/Controller.cs

```diff
@@ -216,12 +216,16 @@ public virtual PartialViewResult PartialView(string viewName, object model)
         /// The view component name. Can be a view component
         /// <see cref="ViewComponents.ViewComponentDescriptor.ShortName"/> or
         /// <see cref="ViewComponents.ViewComponentDescriptor.FullName"/>.</param>
-        /// <param name="arguments">The arguments to pass to the view component.</param>
+        /// <param name="arguments">
+        /// An anonymous <see cref="object"/> containing arguments to be passed to the invoked view component method.
```

Comment:
Ack, this is **false**! There's no such thing as an **anonymous object**!!! There is only an anonymous **type**.

The parameters here are **objects** with properties representing the arguments of the invocation. Whether the caller uses an object of an anonymous type or not is irrelevant. It is 100% perfectly reasonable to use a named-type (or *onymous*, if you will), e.g. `return ViewComponent("MovieDetails", new MovieDetailInfo { id=123, Title="Great Movie", ... });`.

Please fix.